iree/samples/vulkan/CMakeLists.txt

```diff
@@ -19,6 +19,9 @@ iree_bytecode_module(
     "simple_mul.mlir"
   CC_NAMESPACE
     "iree::samples::vulkan"
+  TRANSLATIONS
+    "-iree-mlir-to-vm-bytecode-module"
+    "-iree-hal-target-backends=vulkan-spirv"
```

Comment:
I think this should be FLAGS instead of TRANSLATIONS - both of these are just flags to the iree-translate binary.
In fact, I think the way to split these up would be to have a TARGET ("vm-bytecode-module" would be the current one, but "cc-source" or "llvm-ir" could be future ones) and FLAGS. But I think FLAGS is good enough for now and it makes it easier when reading the rule to know how to break that down and reproduce it yourself ("oh, it's iree-translate with -iree-mlir-to-vm-bytecode-module and ...."). Less magic and easier to search for :)

Reply:
ugh realized this relates to #923

Reply:
Renamed to FLAGS. You can see the diff on https://github.com/google/iree/pull/923. I rebased this on the new commit.